src/pubtools/_marketplacesvm/tasks/delete/command.py

```diff
@@ -135,6 +135,15 @@ def _convert_provider_name(self, provider_name: str) -> List[str]:
         }
         return accounts[provider_name]
 
+    def _set_ami_invisible(self, push_item: AmiPushItem) -> None:
+        img_id = push_item.image_id
+        provider = push_item.marketplace_entity_type
+        log.debug("Marking AMI %s as invisible on RHSM for the provider %s.", img_id, provider)
+        try:
+            self.set_ami_invisible_rhsm(push_item, provider)
+        except Exception as err:
+            log.warning("Failed to mark %s invisible on RHSM: %s", img_id, err)
```

Comment:
I would also set `stack_info=True, exc_info=True` to the warning.
just to provide a bit more info for the troubleshooting.

Reply:
Added

Reply:
TYVM